docs/monitors/http.md

```diff
@@ -92,10 +95,9 @@ dimensions may be specific to certain metrics.
 
 | Name | Description |
 | ---  | ---         |
+| `last_url` | Last URL retrieved (after redirects) from configured one. Only sent if `noRedirects: false` and `addLastURL: true` and if URL responds with a redirect different from the original `url`. |
```

Comment:
I would change everywhere it is called the "last" URL to be the "redirect" URL (e.g. `redirect_url`) instead.  This makes it immediately clear what it represents, at least clearer than "last" is.

Reply:
should be good now